### ISSUE TITLE
using Intersect Space Geometry user script causes SketchUp to crash

### ISSUE THREAD

**Issue by mbethsch** [openstudiocoalition/openstudio-sketchup-plugin#86]
Was having some trouble intersecting & matching surfaces (walls would randomly disappear and some surfaces wouldn't split) so decided to try the Intersect Space Geometry user script. As soon as the script is clicked SketchUp exits and says 'SketchUp has produced a bug splat'. 

Using openstudio-sketchup-plugin v1.2.0 and Sketchup 2021 Pro

**Comment by macumber** [comment]
Thanks for the report, is there any chance you could attach a copy of the osm (pre-intersect) that causes this issue?

**Comment by mbethsch** [comment]
yes -- good catch. I'm attaching it here as a .txt file since .osm doesn't seem to be a supported filetype.
[openstudio_3.txt](https://github.com/openstudiocoalition/openstudio-sketchup-plugin/files/6813354/openstudio_3.txt)

**Comment by macumber** [comment]
I am able to reproduce this, I don't know why it is crashing but there is something off with Space 125 and Space 128
